pkg/api/v2/types.go

```diff
@@ -451,9 +451,10 @@ type ExternalClusterCloudSpec struct {
 
 type KubeOneSpec struct {
 	// Manifest Base64 encoded manifest
-	Manifest  string            `json:"manifest"`
-	SSHKey    KubeOneSSHKey     `json:"sshKey"`
-	CloudSpec *KubeOneCloudSpec `json:"cloudSpec"`
+	Manifest               string            `json:"manifest"`
+	SSHKey                 KubeOneSSHKey     `json:"sshKey"`
+	ContainerRuntimeDocker bool              `json:"containerRuntimeDocker"`
```

Comment:
it should be a string,  like for the KKP Cluster object. It will support other container runtimes in the future